institutions/ipeds.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import unicode_literals
+
+import six
+import csv
+import codecs
+import logging
+
+from schema_transformer.transformer import CSVTransformer
+
+from .institutions import Institution
+
+logger = logging.getLogger(__name__)
+
+
+class IpedsTransformer(CSVTransformer):
+    def _transform_string(self, val, doc):
+        val = super(IpedsTransformer, self)._transform_string(val, doc)
+        return six.u(val)
+
+    def load(self, doc):
```

Comment:
A ` CSVTransformer ` raises an ` ImplementationError ` for a not defined ` load ` function.